.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -27,17 +27,32 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
 
+      - name: Setup node
+        uses: actions/setup-node@v3
+
+      - name: Install docs build dependencies
+        run: npm install -g @mermaid-js/mermaid-cli
+
+      - name: Prepare Mermaid assets for docs
+        run: |
+          for file in assets/*.mmd; do
+            mmdc -i "$file" -o "${file/.mmd/.png}"
```

Comment:
**nitpick (blocking):** Can we swap the output file to SVG for better dark mode compatibility?

Reply:
Good call, done in [9caa3ea](https://github.com/ccao-data/data-architecture/pull/102/commits/9caa3eab66900a129b06f0b5b4f5635c40950e94)! The SVG is live on the [production site](https://ccao-data.github.io/data-architecture) if you want to spot check it.